addons/website/controllers/main.py

```diff
@@ -366,10 +366,11 @@ def get_suggested_link(self, needle, limit=10):
         for name, url, mod in current_website.get_suggested_controllers():
             if needle.lower() in name.lower() or needle.lower() in url.lower():
                 module_sudo = mod and request.env.ref('base.module_%s' % mod, False).sudo()
-                icon = mod and "<img src='%s' width='24px' height='24px' class='mr-2 rounded' /> " % (module_sudo and module_sudo.icon or mod) or ''
+                icon = mod and '%s' % (module_sudo and module_sudo.icon or mod) or ''
```

Comment:
Didn't test the PR but just tried this one and seems wrong:
Before:
![image](https://github.com/odoo/odoo/assets/30048408/db8ebb7f-1bc9-4316-9a9f-e7e54eda8ce2)
After:
![image](https://github.com/odoo/odoo/assets/30048408/2473f162-e702-4310-9fa9-f3f6b6bc7116)